db/migrate/20181207011115_downcase_custom_emoji_domains.rb

```diff
@@ -0,0 +1,5 @@
+class DowncaseCustomEmojiDomains < ActiveRecord::Migration[5.2]
+  def change
+    CustomEmoji.update_all('domain = lower(domain)')
```

Comment:
Use `in_batches.update_all` to prevent table lock ups

Reply:
Awesome, I didn't know this existed!